app/src/main/res/drawable/ic_arrow_forward_black_24dp.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+        android:width="24dp"
```

Comment:
change the name of files also from black to white. It creates a new confusion later